flows/vision-zero/prefect_agent_environment_variables.env_template

```diff
@@ -16,7 +16,6 @@ DB_HOST=
 DB_USER=
 DB_PASS=
 DB_NAME=
-DB_IMPORT_SCHEMA=
```

Comment:
This is dynamic now, and handled in the state-chain passed down through the `.map()`'d functions